topics/admin/tutorials/poll-ssa/slides.html

```diff
@@ -0,0 +1,433 @@
+- 9 respondents from the [Galaxy SSA group](https://galaxyproject.org/community/sig/small-scale-admins/)
+- Questions about various time burdens and technological choices
+- [Full version of the report, + form export](https://hal.science/hal-04491929) 
+- [Raw data (anonymised) + analysis script](https://entrepot.recherche.data.gouv.fr/dataset.xhtml?persistentId=doi:10.57745/SQMQP1)
+]
+
+---
+
+
+### Active Users
+
+
+.pull-left[![Active Users](../../images/poll-ssa/01-Active-Users.jpg)]
+
+
+.pull-right[
+- 3 categories : <10, circa 20 & circa 50
+- Active vs signed up: no problem having a lot more signed up than active users, no need to delete inactive users (in case they come back)
+]
+
+---
+
+
+### DB Tech
+
+
+.pull-left[![DB Tech](../../images/poll-ssa/02-DB-Tech.png)]
+
+
+.pull-right[
+- Necessarily PostgreSQL
+]
+
+
+---
+
+
+### Computing Method
+
+
+.pull-left[![Computing Method](../../images/poll-ssa/03-Computing-Method.png)]
+
+
+.pull-right[
+- Multiple batch schedulers (usually HTCondor), sometimes with Pulsar, at least one on a separate machine from the batch scheduler, at least one on the same.
+]
+
+
+---
+
+
+### Object Store Backend
+
+
+.pull-left[![Object Store Backend](../../images/poll-ssa/04-Object-Store-Backend.png)]
+
+
+.pull-right[
+- Usually local &/or NAS, a S3.
+]
+
+
+---
+
+
+### Gravity Usage
+
+
+.pull-left[![Gravity Usage](../../images/poll-ssa/05-Gravity-Usage.png)]
+
+
+.pull-right[
+- About 50/50
+- Recent versions seem to include it implicitly
+]
+
+
+---
+
+
+### End-User Support Burden
+
+
+.pull-left[![End User Support Burden](../../images/poll-ssa/06-End-User-Support-Burden.jpg)]
+
+
+.pull-right[
+- 1 hour per week average end-user support
+- Lots of training (though that’s part of the next question)
+- Lots of developing &/or debugging users’ tools/workflows for them.
+]
+
+
+---
+
+
+### User Training Burden
+
+
+.pull-left[![User Training Burden](../../images/poll-ssa/07-User-Training-Burden.jpg)]
+
+
+.pull-right[
+- 1 hour per week average user training
+- Usually makes use of existing Galaxy tutorials, sometimes in-person
+- The tutorials aren’t always specific enough, requiring some custom Q&A
+]
+
+
+---
+
+
+### .pull-right[User Tool Dev & Planemo Usage]
+
+.pull-left[
+.image-50[![User Tool Dev Local](../../images/poll-ssa/08A-User-Tool-Dev-Local.png)]
+.image-50[![User Tool Dev Planemo](../../images/poll-ssa/08B-User-Tool-Dev-Planemo.png)]
+]
+
+
+.pull-right[
+
+ ⠀
+
+ ⠀
+
+ ⠀
+
+- Users developing their own tools is rare, and if they do, they rarely use Planemo, which I found surprising since I thought it was used to simplify tool dev.
+- It turns out tool creation is mostly done through copy/pasting XMLs
+- Planemo shines when it comes to tool testing, linting and publishing
+]
+
+???
+Tried to find a vertical align CSS class, but .center did not work, nor did <div style="vertical-align:middle;"></div>
+So ended up putting an invisible braille character
```

Comment:
I have fixed this in a different way which I will push to this branch.

Reply:
Thanks, I saw your fix, much better.